doc/howto/network_ovn_setup.md

```diff
@@ -177,3 +177,35 @@ See the linked YouTube video for the complete tutorial using four machines.
        ping <IP of c1>
        ping <nameserver>
        ping6 -n www.example.com
+
+## Send OVN logs to LXD
+
+Complete the following steps to have the OVN controller send its logs to LXD.
+
+1. Enable the syslog socket:
+
+       lxc config set core.syslog_socket=true
+
+1. Open `/etc/default/ovn-host` for editing.
+
+1. Paste the following configuration
+
+       OVN_CTL_OPTS=" \
+              --ovn-controller-log='-vsyslog:info --syslog-method=unix:/var/snap/lxd/common/lxd/syslog.socket'"
+
+1. Restart OVN controller:
+
+       systemctl restart ovn-controller.service
+
+1. Logs from the OVN controller can now be seen using the monitor:
+
+       lxc monitor --type=ovn
+
+1. If the logs are to be sent to Loki, the `ovn` value needs to be added to the `loki.types` configuration key.
+
+1. The same can be achieved for OVN northd, OVN NB ovsdb-server, and OVN SB ovsdb-server. In this case `/etc/default/ovn-central` needs to be edited.
+
+       OVN_CTL_OPTS=" \
+	       --northd-log='-vsyslog:info --syslog-method=unix:/var/snap/lxd/common/lxd/syslog.socket' \
+	       --nb-log='-vsyslog:info --syslog-method=unix:/var/snap/lxd/common/lxd/syslog.socket' \
+	       --sb-log='-vsyslog:info --syslog-method=unix:/var/snap/lxd/common/lxd/syslog.socket'"
```

Comment:
````suggestion
You can now use [lxc monitor](lxc_monitor.md) to see logs from the OVN controller:

    lxc monitor --type=ovn

You can also send the logs to Loki.
To do so, add the `ovn` value to the {config:option}`server-loki:loki.types` configuration key.

```{tip}
You can include logs for OVN `northd`, OVN north-bound `ovsdb-server`, and OVN south-bound `ovsdb-server` as well.
To do so, edit `/etc/default/ovn-central`:

    OVN_CTL_OPTS=" \
       --northd-log='-vsyslog:info --syslog-method=unix:/var/snap/lxd/common/lxd/syslog.socket' \
       --nb-log='-vsyslog:info --syslog-method=unix:/var/snap/lxd/common/lxd/syslog.socket' \
       --sb-log='-vsyslog:info --syslog-method=unix:/var/snap/lxd/common/lxd/syslog.socket'"
```
````

Reply:
This part shouldn't be in the steps - it's the result and additional information.